css-brochure-form/styles.css

```diff
@@ -0,0 +1,77 @@
+
+* {
+  font-family: "Brandon Grotesque", Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  background-color: #fff;
+}
+
+h3 {
+  text-align: center;
+  color: #ec615b;
+  margin-bottom: 8px;
+  text-transform: uppercase;
+  font-family: "Brandon Grotesque", Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+}
+
+div.div1 {
+  width: 37%;
+  margin: auto;
+}
+
+div.div2 {
+  text-align: right;
+  width: 37%;
+  margin: auto;
+}
```

Comment:
These class names aren't very descriptive of their purpose, try to use a name that better articulates what they are for.